### DISCUSSION TITLE
Livecode not working

### DISCUSSION THREAD

**Discussion by Phys-Prof-Nielsen** [executablebooks/meta#1156] [Q&A]
I have a jupyter book that has been using livecode (thebe). It recently stopped working. Any suggestions to what may have happened. I have not made any changes to my book. I tried a few other jupyterbooks in the example gallery that are using livecode and neither of those were working either.

**Comment by choldgraf** [comment]
I'd check the JavaScript logs in the console of the browser you're using. That often reveals some information about what's going on. It's often because the public mybinder.org service is not being reliable. That service has limited resources behind it so it occasionally suffers from performance drops.
